Automattic/wp-calypso
# Site Settings: Remove unreleased theme setup interface

#### Changes proposed in this Pull Request

The theme setup interface never got it to staging or production - it only got to wpcalypso in #11914. That was almost 5 years ago, and nothing has changed ever since. I think it's time to remove that interface already, together with state and feature flag.

Question: should we maintain a redirect from `/settings/theme-setup` to `/settings/general`, or since the feature never landed, we can actually remove that route completely?

#### Testing instructions

* Verify the removed code is not in use.
* Verify Calypso builds properly and tests still pass.
* Go to `/settings/theme-setup` and verify you're redirected to `/settings/general`
* Go to `/settings/theme-setup/:site` (where `:site` is one of your sites) and verify you're redirected to `/settings/general/:site`.
* Go to `/settings/general/:site` (where `:site` is one of your sites) and verify the Site Tools menu looks well.
